src/main/java/com/synopsys/integration/detect/lifecycle/run/step/IntelligentModeStepRunner.java

```diff
@@ -21,19 +19,14 @@
 import com.synopsys.integration.detect.lifecycle.run.operation.OperationRunner;
 import com.synopsys.integration.detect.lifecycle.run.operation.blackduck.BdioUploadResult;
 import com.synopsys.integration.detect.lifecycle.run.step.utility.StepHelper;
-import com.synopsys.integration.detect.tool.iac.IacScanCodeLocationData;
 import com.synopsys.integration.detect.tool.impactanalysis.service.ImpactAnalysisBatchOutput;
-import com.synopsys.integration.detect.tool.signaturescanner.SignatureScannerCodeLocationResult;
 import com.synopsys.integration.detect.util.filter.DetectToolFilter;
 import com.synopsys.integration.detect.workflow.bdio.BdioResult;
-import com.synopsys.integration.detect.workflow.blackduck.codelocation.CodeLocationAccumulator;
-import com.synopsys.integration.detect.workflow.blackduck.codelocation.CodeLocationResults;
```

Comment:
Without CodeLocationAccumulator, I'm not sure how the MaaS code (branch sb-correlation-id-and-counts) will get the list of created codelocations. Is there a new/alternate mechanism for collecting them for MaaS?

Reply:
@stevebillings No, there isn't an alternate mechanism. I'm not aware of what work you might be doing for MaaS. The CodeLocationAccumulator file exists again with the latest changes if you need to make use of it for your MaaS work. Or are you suggesting that the usage of it be restored here in IntelligentModeStepRunner. If that is the case then I'm not sure we can really remove any code. The bom-status endpoint provides a way for us to avoid waiting on the code locations but if something else is going to need it then we can't pull any of this logic, no? Should I restore it to be safe and only keep the new bom-status work in this MR? Thanks for thoughts.

Reply:
It's up to you and @gopannabd , but you might want to merge/test the two branches together. You can always delete the new MaaS property to keep the feature hidden until Black Duck is ready.

Reply:
At this point I feel most comfortable restoring the code. The MR should now reflect the original state that just deals with the bom-status endpoint.